### ISSUE TITLE
[Frontend] Notifications for unconfirmed periods/weeks

### ISSUE THREAD

**Issue by trondoe** [Puzzlepart/did#254]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Add notifications support for unconfirmed weeks in Did365.

####  What is the feature?
The feature provides notifications to Did365 users who have unconfirmed weeks. This will hopefully encourage them to confirm weeks, and maybe even add better routines when it comes to confirming their hours.
#### Why should we add this feature?
To improve the percentage of employees who confirm their weeks, and thereby ensure that the numbers are more up-to-date. 
Also, to avoid that leaders have to spend their time sending friendly reminders about missing week confirmations.
#### How does this improve the product?
I believe it does. Hence the "Why should we add this feature" section above.

### TODOS
- [x] Graphql endpoint for notifications
- [x] Sexyness on toggle, icon etc
- [x] i18n for month name
- [x] Support split weeks, message example: "Week 18 (May) is not confirmed."
